sites/all/modules/features/challenges/challenges.module

```diff
@@ -203,6 +203,9 @@ function challenges_preprocess_node(&$variables) {
       $variables['challenge_statistics'] = implode(', ', $challenge_statistics);
     }
 
+    $author = user_load($variables['node']->uid);
+    $variables['submitted_by'] = 'Submitted on ' . format_date($variables['created'], 'custom', 'd/m/Y') . ' by ' . l($author->name, 'user/' . $author->uid);
```

Comment:
This is not needed, if you check ' Display author and date information.' in challenge content type display setting you will get this from core as $variables['submitted']